src/Uno.UI/UI/Xaml/UIElement.Skia.cs

```diff
@@ -16,23 +18,20 @@ namespace Windows.UI.Xaml
 {
 	public partial class UIElement : DependencyObject
 	{
-		// Even if this a concept of FrameworkElement, the loaded state is handled by the UIElement in order to avoid
-		// to cast to FrameworkElement each time a child is added or removed.
-		internal bool IsLoaded;
-
 		internal Size _unclippedDesiredSize;
 		internal Point _visualOffset;
-		internal List<UIElement> _children = new List<UIElement>();
 		private ContainerVisual _visual;
 		private Visibility _visibilityCache;
 		internal double _canvasTop;
 		internal double _canvasLeft;
 		private Rect _currentFinalRect;
 
-		private protected int? Depth { get; private set; }
-
 		public UIElement()
 		{
+			_log = this.Log();
+			_logDebug = _log.IsEnabled(LogLevel.Debug) ? _log : null;
+			_isFrameworkElement = this is FrameworkElement;
```

Comment:
Codacy found an issue: [Offload the code that's conditional on this 'is' test to the appropriate subclass and remove the test.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5975891)